samples/browser/build.gradle

```diff
@@ -76,7 +76,13 @@ android {
             setIgnore(true)
         }
 
-        if (flavors.contains("gecko") && flavors.contains("universal")) {
+        // Only allow "universal" flavors of "system" and "geckoviewNightly" engines
+        if (flavors.contains("universal") && !(flavors.contains("system") || flavors.contains("geckonightly"))) {
```

Comment:
Is this case-sensitive? Does `geckonightly` need to be `geckoNightly`?

Reply:
I was wondering about this too, but we join+lowercase above:
`def flavors = variant.flavors*.name.toString().toLowerCase()`